tests/test_survey_schema_translation.py

```diff
@@ -174,14 +175,13 @@ def test_get_catalog():
                             "guidance": {
                                 "hide_guidance": "Hide feeling answer help",
                                 "show_guidance": "Show feeling answer help",
-                                "content": [
+                                "contents": [
```

Comment:
Is this simply a change not yet migrated within this test?

Reply:
Yes. It didn't matter previously as we just extracted all title, description etc elements. Now we specify what property they belong to, it highlighted that `content` should always have been `contents`.